Bottom: b789541152cc00f83febf350e5c8844d43d04a12
Top:    50dab70ad7eb3dbdef71c75d4d93c9cb91e57f58
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:54:49 +0000

Refresh of fix-return-codes-from

---

diff --git a/src/polygon1.c b/src/polygon1.c
index f476cc4..1dd9607 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2990,7 +2990,8 @@ poly_Copy0 (POLYAREA ** dst, const POLYAREA * src)
 {
   *dst = NULL;
   if (src == NULL)
-    return TRUE; /* Copying a NULL POLYAREA is not a failure, return true */
+    return TRUE;
+
   *dst = (POLYAREA *)calloc (1, sizeof (POLYAREA));
   if (*dst == NULL)
     return FALSE;
@@ -3020,7 +3021,7 @@ poly_M_Copy0 (POLYAREA ** dst, const POLYAREA * srcfst)
 
   *dst = NULL;
   if (src == NULL)
-    return TRUE; /* Copying a NULL POLYAREA is not a failure, return true */
+    return TRUE;
 
   do
     {
